Header header
int32 thrust_in_counts
float64[3] accel_inertial
float64[3] position